ci-jobs/templates/xcuitest-e2e-template.yml

```diff
@@ -19,7 +20,7 @@ jobs:
       vmImage: ${{ parameters.vmImage }}
       script: |
         npm install --no-save mjpeg-consumer
-        npm run e2e-test:basic -- --grep "XCUITestDriver - basics -"
+        npm run e2e-test:basic -- --grep "XCUITestDriver - alerts -"
```

Comment:
You can add a new line for now instead of modifying basics to alerts (to run all of them)

Reply:
what is the purpose of this grep arg?

Reply:
> You can add a new line for now instead of modifying basics to alerts (to run all of them)

thanks, done

> what is the purpose of this grep arg?

I would like to run only fixed tests.

Reply:
maybe then it would make sense to add a new task(s) under package.json if you want to categorize these tests differently